firestore/angular-rewrite/README.md

```diff
@@ -1,27 +1,55 @@
-# FriendlyEatsQuickstart Angular
-
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 16.1.3.
-
-## Running the Application with Emulators
-
-The command to start the project and run the local emulators is:
-`firebase --project demo-friendly-eats emulators:exec --import./exported-firestore [--export-on-exit=./exported-firestore]`
-(Only the `--export-on-exit` flag only if you want to persist changes after each run of the emulators. Otherwise, you will get a fresh dataset each execution.)
-
-## Running the Angular App Only
-
-Run `ng serve` to serve the Angular frontend without the emulated Firebase backend. Navigate to `http://localhost:4200/`. The application will automatically reload if you change any of the source files.
-
-## Code scaffolding
-
-Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
-
-## Build
-
-Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory.
-
-## Further help
-
-To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI Overview and Command Reference](https://angular.io/cli) page.
-
-
+# Firestore Quickstart
+
+## Introduction
+
+FriendlyEats is a restaurant recommendation app built on Firestore. It was built with the [Angular](https://angular.io/) Javascript framework, and makes use of [Firebase's Local Emulator Suite](https://firebase.google.com/docs/emulator-suite) so that the quickstart can be run without needing to create a project in the [Firebase Console](https://console.firebase.google.com).
```

Comment:
Did we get any tech writer to check this out by any chance?

Reply:
Good point, we should discuss with @markarndt